### PR TITLE
added pretty handle example

### PR DESCRIPTION
**Pull request by SterlingButters**

### CODE REVIEW

**Comment by SterlingButters** [comment]
@xhlulu just realized that new example doesn't work correctly for moving while graph is displayed, will have to fix that but regardless its working more or less

UPDATE: Fixed in 2nd commit - let me know what you think

**Comment by xhluca** [comment]
Thank you for adding the example! Could you share a screenshot of the example?

Also could you elaborate on what the "acorn" library do?

**Comment by SterlingButters** [comment]
Here is an example:
ezgif com-video-to-gif

I guess I forgot to show that you can also drag it while the graph is hidden but you can haha

And honestly idk where the acorn library came from - feel free to remove it

File: dash_draggable/package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "author": "Xing Han Lu xinghan@plot.ly",
     "license": "MIT",
     "dependencies": {
+        "acorn": "^6.1.0",
```
Comment (xhluca) [inline dash_draggable/package.json:18]: Could you remove this please? that way I won't need to make a new PR :)

File: package.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
     "author": "Xing Han Lu xinghan@plot.ly",
     "license": "MIT",
     "dependencies": {
+        "acorn": "^6.1.0",
```
Comment (xhluca) [inline package.json]: Same as above

**Review by xhluca — APPROVED** [review]
The gif looks awesome! Thank you for adding this. Once you remove acorn as a dependencies (or keep it, if it's needed for anything), feel free to merge it!

File: dash_draggable/package.json (hunk 1 above)
Comment (SterlingButters) [reply dash_draggable/package.json:18]: Good Point

**Pull request merged**